admin/app/router.js

```diff
@@ -92,6 +92,9 @@ Router.map(function () {
 
     this.route('complementary-certifications', function () {
       this.route('list');
+      this.route('complementary-certification', { path: '/:complementary_certification_id' }, function () {
```

Comment:
Un simple id ne suffirait pas?
```suggestion
      this.route('complementary-certification', { path: '/:id' }, function () {
```

Reply:
Ça ne marche pas si on indique juste id

<img width="1523" alt="Capture d’écran 2023-07-20 à 19 02 48" src="https://github.com/1024pix/pix/assets/58915422/0ec08fea-5808-43e9-b9ca-7dde58b1cd5b">